src/dp_transformers/grad_sample/transformers/tied_embedding.py

```diff
@@ -38,8 +40,8 @@ def compute_tied_embedding_grad_sample(
     grad_sample.scatter_add_(1, index, B.reshape(batch_size, -1, layer.embedding_dim))
     torch.backends.cudnn.deterministic = saved
 
-    utils.create_or_accumulate_grad_sample(layer.weight, grad_sample, layer)

```

Comment:
Before Opacus 1.0 they had this function `create_or_accumulate_grad_sample` to add two or more gradients coming from tied parameters. New version removed this function but then they fixed it with some other approach https://github.com/pytorch/opacus/issues/413. This change would not add up to the gradients but actually replace it so it would not be right. We can remove this script altogether as it's being handled automatically now afaik :)